### ISSUE TITLE
REPORT: Audio handoff not working

### ISSUE THREAD

**Issue by bricker** [SCPR/KPCC-Android#98]
Any time I am listening and I do something that overtakes audio (like click on a link in an email or app that opens youtube, or an internet video ad) and occasionally when I get a notification from my phone, the KPCC app will not resume audio.>>I can hit [start from beginning], [go to live].. rewind fast forward... no audio, the only fix is to open [programs] choose any other program, then I get audio, then go back and choose the live feed or the program I was listening to. This happens while live or listening to a pre-aired program.

**Comment by vsinghkpcc** [comment]
Also can't consistently get this to fail, though when I did this seems to be the process:

1. Play the live stream in the KPCC App.
2. Navigate to a different app and play audio (I played a FB embedded video on scpr.org). KPCC Audio should stop and the "new" audio should start playing.
3. Return to the KPCC App and hit the play button. Note that the other audio will stop, but no audio from KPCC plays (the two times I got this to fail I waited for probably 30 seconds without hearing audio from the KPCC App)
